README.md

```diff
@@ -195,14 +195,43 @@ $ remme public-key get-list \
       --node-url=node-genesis-testnet.remme.io
 {
     "result": {
-        "public_key_addresses": [
+        "addresses": [
             "a23be10b3aad1b4a98f338c71d6dcdb2aa2f296c7e31fb400615e335dc10dd1d4f62bf",
             "a23be14b362514d624c1985277005327f6fc40413fb090eee6fccb673a32c9809060ff"
         ]
     }
 }
 ```
 
+Get information about public key by public key address — ``remme public-key get-info``:
```

Comment:
Consider to change it to `Get information about public key by its address`. A bit easier to read.

Reply:
@dmytrostriletskyi, done.